e3sm_to_cmip/cmor_handlers/utils.py

```diff
@@ -73,7 +73,7 @@ def load_all_handlers(
 
             handlers = handlers + var_handler
 
-        if len(missing_handlers) > 0:
+        if len(missing_handlers) == len(cmip_vars):
             raise KeyError(
                 f"No handlers are defined for the variables: {missing_handlers}. "
                 "Make sure at least one variable handler is defined for each of these "
```

Comment:
```suggestion
        if len(missing_handlers) == len(cmip_vars):
            logger.warning(
                f"No handlers are defined for the variables: {missing_handlers}. "
                "Make sure at least one variable handler is defined for each of these "
```

I think the conditional should be kept as `if len(missing_handlers) > 0:` because **it indicates that there is at least 1 variable without a defined handler and prints all variables with missing handlers.** 

We can print a `logger.warning` instead of raising a `KeyError` if we want to continue processing variables with defined handlers.